src/jit/lower.cpp

```diff
@@ -108,20 +108,39 @@ bool Lowering::IsSafeToContainMem(GenTree* parentNode, GenTree* childNode)
 // IsContainableMemoryOp: Checks whether this is a memory op that can be contained.
 //
 // Arguments:
-//    node - the node of interest.
+//    node        - the node of interest.
+//    useTracked  - true if this is being called after liveness so lvTracked is correct
+//
+// Return value:
+//    True if this will definitely be a memory reference that could be contained.
 //
 // Notes:
 //    This differs from the isMemoryOp() method on GenTree because it checks for
-//    the case of an untracked local. Note that this won't include locals that
-//    for some reason do not become register candidates, nor those that get
+//    the case of doNotEnregister local. This won't include locals that
+//    for some other reason do not become register candidates, nor those that get
 //    spilled.
+//    Also, if we call this before we redo liveness analysis, any new lclVars
+//    introduced after the last dataflow analysis will not yet be marked lvTracked,
+//    so we don't use that.
 //
-// Return value:
-//    True if this will definitely be a memory reference that could be contained.
-//
-bool Lowering::IsContainableMemoryOp(GenTree* node)
+bool Lowering::IsContainableMemoryOp(GenTree* node, bool useTracked)
```

Comment:
Minor nit: do you think that callers will always pass a constant argument for the `useTracked` parameter (I think that's the case in this change)? If so, you might consider making it a template parameter.